Bump github/codeql-action from 3.28.3 to 3.28.9 (#258)
rsksmart/devportal: develop/apps/wallets/nifty.md

---
layout: rsk
title: Nifty

> ---

Reviewer: ```suggestion
--- # How To Create A Nifty Wallet
```

> 

Reviewer: ```suggestion
## Installing Nifty Wallet
```

> For Nifty installation, please <a href="https://www.poa.network/for-users/nifty-wallet">visit here</a>.

Reviewer: ```suggestion
To install the Nifty wallet, please visit [POA Nifty](https://www.poa.network/for-users/nifty-wallet)
```
